lib/js/src/manager/lifecycle/LifecycleManager.js

```diff
@@ -259,9 +259,9 @@ class LifecycleManager {
                 case FunctionID.RegisterAppInterface:
                     this._processRaiResponse(rpcMessage);
                     break;
-                case FunctionID.OnHMIStatus:
+                case FunctionID.OnHMIStatus: {
                     // send a single onProxyConnected, when we go from a null HMI level to a defined HMI level
-                    const shouldInit = rpcMessage.getHMILevel() !== null
+                    const shouldInit = rpcMessage.getHMILevel() !== null 
```

Comment:
Is this space being added a part of another rule?

Reply:
oh, interesting. I'm not sure where that came from. Current linter settings don't seem to pick that up as something to be standardized. I'll remove that space for now